src/parser.rs

```diff
@@ -1159,6 +1159,33 @@ impl<'a> Parser<'a> {
                     }
                 }
                 Keyword::XOR => Some(BinaryOperator::Xor),
+                Keyword::OPERATOR if dialect_of!(self is PostgreSqlDialect) => {
+                    debug!("parsing custom operator");
```

Comment:
```suggestion
```

Not sure that will add much after the initial implementation

Reply:
Good call, I have removed this.